Components/Queues/Drivers/StompQueueDriver.php

```diff
@@ -386,7 +386,7 @@ public function nack()
         $this->currentFrame = null;
     }
 
-    public function createQueueMessage()
+    public function createQueueMessage($queueName = null, $options = [])
```

Comment:
The parameters `$queueName` and `$options` can be deleted from the method definition because they are not used

Reply:
Done.